frontend/javascripts/oxalis/view/right-border-tabs/dataset_info_tab_view.tsx

```diff
@@ -568,148 +527,142 @@ export class DatasetInfoTabView extends React.PureComponent<Props, State> {
+      <Tooltip
+        title={
+          <div>
+            Rendered magnification per layer:
+            <ul>
+              {Object.entries(activeMagOfEnabledLayers).map(([layerName, mag]) => {
+                const readableName = getReadableNameForLayerName(dataset, annotation, layerName);
+
+                return (
+                  <li key={layerName}>
+                    {readableName}: {mag ? mag.join("-") : "none"}
+                  </li>
+                );
+              })}
+            </ul>
+            Available resolutions:
+            <ul>
+              {resolutionUnion.map((mags) => (
+                <li key={mags[0].join()}>{mags.map((mag) => mag.join("-")).join(", ")}</li>
+              ))}
+            </ul>
+          </div>
+        }
+        placement="left"
+      >
+        <tr>
+          <td
+            style={{
+              paddingRight: 4,
+              paddingTop: 8,
+            }}
+          >
+            <img
+              className="info-tab-icon"
+              src="/assets/images/icon-downsampling.svg"
+              alt="Resolution"
+            />
+          </td>
+          <td
+            style={{
+              paddingRight: 4,
+              paddingTop: 8,
+            }}
+          >
+            <Typography.Text type="secondary">
               {representativeResolution.join("-")}
               {isActiveResolutionGlobal ? "" : "*"}{" "}
-            </td>
-          </tr>
-        </Tooltip>
-      ) : null;
+            </Typography.Text>
+          </td>
+        </tr>
+      </Tooltip>
+    ) : null;
+  }
+
+  render() {
+    const { dataset } = this.props;
+
     return (
-      <div className="flex-overflow padded-tab-content">
-        <div className="info-tab-block">
-          {this.getTracingName()}
-          {this.getDatasetName()}
-          {this.maybePrintOwnerAndContributors()}
-        </div>
+      <div
+        className="flex-overflow padded-tab-content"
+        style={{ paddingLeft: 20, paddingRight: 20 }}
+      >
+        {this.getAnnotationName()}
+        {this.getAnnotationDescription()}
+        {this.getDatasetName()}
+        {this.maybePrintOwnerAndContributors()}
 
         <div className="info-tab-block">
+          <p className="sidebar-label">Dimensions</p>
           <table
             style={{
               fontSize: 14,
+              marginLeft: 4,
             }}
           >
             <tbody>
               {this.maybePrintOrganization()}
               <VoxelSizeRow dataset={dataset} />
               <DatasetExtentRow dataset={dataset} />
-              {resolutionInfo}
-
-              {features().jobsEnabled &&
-              activeUser != null &&
-              (activeUser.isDatasetManager || activeUser.isAdmin)
-                ? this.getProcessingJobsMenu()
-                : null}
+              {this.getResolutionInfo()}
             </tbody>
           </table>
-          {this.renderSelectedStartingJobsModal()}
```

Comment:
so, these actions can only be triggered from an annotation (instead of from the dataset view) now, right?

Reply:
Good catch. That was not intentional. I think this menu entry should be available in both situations. Will fix.